mex/extractors/datenkompass/load.py

```diff
@@ -0,0 +1,48 @@
+import json
+from collections.abc import Iterable
+
+import boto3
+from botocore.client import BaseClient
+
+from mex.common.logging import logger
+from mex.extractors.datenkompass.item import AnyDatenkompassModel
+from mex.extractors.settings import Settings
+
+
+def start_s3_client() -> BaseClient:
+    """Start up S3 session."""
+    settings = Settings.get()
+    session = boto3.Session(
+        aws_access_key_id=settings.s3_access_key_id.get_secret_value(),
+        aws_secret_access_key=settings.s3_secret_access_key.get_secret_value(),
+    )
+
+    return session.client("s3", endpoint_url=str(settings.s3_endpoint_url))
+
+
+def write_item_to_json(
+    datenkompassitems: Iterable[AnyDatenkompassModel],
```

Comment:
```suggestion
    datenkompassitems: list[AnyDatenkompassModel],
```
should be okay, right?